lib/types/urlencoded.js

```diff
@@ -188,13 +188,13 @@ function createQueryParser (options, extended) {
  * Get the charset of a request.
  *
  * @param {object} req
+ * @returns {string|undefined}
```

Comment:
I'm not sure if it's necessary, but at https://jsdoc.app/tags-returns, it recommends that the return be in the following format:
```suggestion
 * @returns {(string|undefined)}
```

Reply:
Hello @bjohansebas , I appreciate your suggestions, they were on point, and I’ve committed the changes.